nab/detectors/goodness_of_fit/goodness_of_fit_detector.py

```diff
@@ -0,0 +1,156 @@
+import numpy
+import math
+from scipy import stats
```

Comment:
imports in alpha order, and separate built-ins from package imports with a blank line